image_tools/test/cam2image.txt

```diff
@@ -0,0 +1,2 @@
+Publishing data on topic 'image'
+Publishing image #1
```

Comment:
launch testing can work with either `.txt` or `.regex`. `.txt` has to match explicitly. `cam2image.txt` will be matched as a literal file because of the particular block that you referenced